whittle/loss/kd_loss.py

```diff
@@ -51,16 +51,22 @@ def forward(self, outputs, labels, outputs_teacher) -> nn.Tensor:
         """
         soft_target_loss = 0
         outputs_teacher = outputs_teacher.detach()
+
         if outputs_teacher is not None and self.distillation_weight > 0:
             soft_target_loss = self.kldiv(
                 F.log_softmax(outputs / self.temperature, dim=1),
                 F.softmax(outputs_teacher / self.temperature, dim=1),
             ) * (self.temperature**2)
 
-        hard_target_loss = F.cross_entropy(outputs, labels, reduction="mean")
+        hard_target_loss = F.cross_entropy(
+            outputs.reshape(-1, outputs.size(-1)), 
```

Comment:
Changing the shape here leads to a crash in the other unit test. Could we revert the change here and instead reshape the output in the distill.py